import java.io.IOException;

public class HomeWorkRunner {
    public static void main(String[] args) throws IOException {
        System.out.println("Задание 4");
        new HomeWork4().homework4();

        System.out.println("Задание 5");
        new HomeWork5().homework5();

        System.out.println("Задание 6");
        new HomeWork6().homework6();

        System.out.println("Задание 7");
        new HomeWork7().homework7();

        System.out.println("Задание 8");
        new HomeWork8().homework8();

        System.out.println("Задание 9");
        new HomeWork9().homework9();

        System.out.println("Задание 10");
        new HomeWork10().homework10();

        System.out.println("Задание 11");
        new HomeWork11().homework11();

        System.out.println("Задание 12");
        new HomeWork12().homework12();

        System.out.println("Задание 13");
        new HomeWork13().homework13();

        System.out.println("Задание 14");
        new HomeWork14().homework14();

        System.out.println("Задание 15");
        new HomeWork15().homework15();

        System.out.println("Задание 16");
        new HomeWork16().homework16();

        System.out.println("Задание 17");
        new HomeWork17().homework17();

        System.out.println("Задание 18");
        new HomeWork18().homework18();

        System.out.println("Последнее задание 1");
        new FinalWork1().finalwork1();

        System.out.println("Последнее задание 2");
        new FinalWork2().finalwork2();

        System.out.println("Последнее задание 3");
        new FinalWork3().finalwork3();
    }
}

